### DISCUSSION TITLE
插件如何读写配置文件？

### DISCUSSION THREAD

**Discussion by the-cold-heart** [nonebot/discussions#194] [问答] [answered]
如果我写了一个插件，这个插件有一些可以修改的配置，我该如何让插件读取配置文件，并且实现通过指令修改配置文件？

**Comment by yanyongyu** [comment, marked as answer]
参考 https://v2.nonebot.dev/docs/next/tutorial/plugin/config-plugin

**Reply by the-cold-heart** [reply]
> 参考 https://v2.nonebot.dev/docs/next/tutorial/plugin/config-plugin

我看过这篇文章，但是实在没看懂该如何操作，而且文章中似乎只写了如何读取，没有写如何写入……

**Reply by yanyongyu** [reply]
没有写入，这些配置项都是没有持久化的

**Reply by the-cold-heart** [reply]
> 没有写入，这些配置项都是没有持久化的

好的，谢谢大佬解惑
